hack/build.sh

```diff
@@ -41,6 +41,12 @@ release)
 	TAGS="${TAGS} release"
 	if test "${SKIP_GENERATION}" != y
 	then
+		if ! $(go list github.com/shurcooL/vfsgen &> /dev/null);
```

Comment:
Hm, looks like we're vendoring it today.  Are you building from `GOPATH`?

Reply:
i'm a bit of a go noob but I believe the `go generate ./data` is not looking into the vendor directory at all.. i.e. it's not a `go build`, it's a separate operation which doesn't appear to be looking in vendor dir.

Reply:
> Are you building from `GOPATH`?

yes, i'm building from `GOPATH`.